### ISSUE TITLE
4852-4854 W Cortez problem with squareFeet value

### ISSUE THREAD

**Issue by wysow** [nmathey/finasync#14]
Hello,

First of all thanks a lot for this tool!

Running the command `poetry run python -m finasync realt rent` is giving me error:

```
INFO:root:add 1.0 4852-4854 W Cortez @ 50.9
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/enoptea/PhpstormProjects/finasync/finasync/__main__.py", line 60, in <module>
    sys.exit(main())
  File "/home/enoptea/PhpstormProjects/finasync/finasync/__main__.py", line 50, in main
    result = sync_realt_rent(
  File "/home/enoptea/PhpstormProjects/finasync/finasync/realt.py", line 342, in sync_realt_rent
    (squareFeet * 0.092903), 0
TypeError: unsupported operand type(s) for *: 'NoneType' and 'float'
```

**Comment by nmathey** [comment]
Hi,

That's because squareFeet data is missing from API for this property and provides a `null` value. I thought having fix this scenario by forcing to 1 to avoid such issue :/

Do you have a working API Key on your side already?
On your `RealT_OfflineTokensList.json` file (cached data from API), which value do you have for squareFeet attribute for this property?

On mine I have O but that is before another fix I did. As a workaround for now maybe set this attribute to 1 or to the info we can find on RealT website. Based on your response I will work on another fix (Finary doesn't accept 0 value neither :) ).

```json
"0xe7b6de709ffc3bd237c2f2c800e1002f97a760f3": {
            "fullName": "4852-4854 W Cortez St, Chicago, IL 60651",
            "shortName": "4852-4854 W Cortez",
            "tokenPrice": 50.9,
            "netRentMonthPerToken": 0.32416666666667,
            "currency": "USD",
            "rentStartDate": {
                "date": "2021-06-09 00:00:00.000000",
                "timezone_type": 3,
                "timezone": "UTC"
            },
            "squareFeet": 0,
            "totalTokens": 18000,
            "totalInvestment": 916200,
            "grossRentMonth": null,
            "propertyManagement": 792,
            "realtPlatform": null,
            "insurance": 265,
            "propertyTaxes": 1334,
            "propertyMaintenanceMonthly": 0,
            "utilities": 1476,
            "netRentMonth": 5835,
            "coordinate": {
                "lat": "41.899828",
                "lng": "-87.748084"
            },
            "propertyType": 2,
            "rentalType": "long_term",
            "productType": "real_estate_rental"
        },
```

**Comment by wysow** [comment]
For me the value is `null`, I submitted a potential fix in #15, let me know if it's ok for you...

**Comment by wysow** [comment]
Just looked on RealT website and real value is `10040` it seems.

**Comment by wysow** [comment]
I just added this value in the realT api cache file in my PR.

**Comment by nmathey** [comment]
> For me the value is `null`, I submitted a potential fix in #15, let me know if it's ok for you...

Yeah I thought my fix was ok but let's use yours: we need to handle this because cached file is updated from API if 7+ days old so null value may reappear with new properties.
I added a quick request for the cached file on the repo: if you can handle it and I'll merge everything.

Thanks

**Comment by nmathey** [comment]
Also I just reported this to the RealT Community team if they can update those missing values in API.
